core/go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.5.0
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.0
+	github.com/jackc/pgx/v5 v5.5.2
```

Comment:
Is it OK if we do upgrade pgx to 5.5.5?  We were planning to do so to resolve a security alert in https://github.com/kwilteam/kwil-db/pull/722.  I noticed a "downgrade" commit in this PR and wasn't sure if you hit a problem or were just aiming to keep the changes minimal.

Reply:
I got some weird build error while upgrading, so I downgraded. Will upgrade though

Reply:
No need, we upgraded in https://github.com/kwilteam/kwil-db/pull/722.
Just wanted to make sure there wasn't something problematic about 5.5.5